content/docs/0.8.1/high-availability/recover-volume.md

```diff
@@ -1,101 +1,101 @@
 ---
-  title: Recover Volume after Unexpected Detachment
+  title: Recovering a Volume after Unexpected Detachment
   weight: 1
 ---
 
-Longhorn can automatically reattach then remount volumes if an unexpected detachment happens, which can happen during a [Kubernetes upgrade](https://github.com/longhorn/longhorn/issues/703) or a [Docker reboot](https://github.com/longhorn/longhorn/issues/686).
+Longhorn can automatically reattach, then remount volumes if an unexpected detachment happens, which can happen during a [Kubernetes upgrade](https://github.com/longhorn/longhorn/issues/703) or a [Docker reboot](https://github.com/longhorn/longhorn/issues/686).
 
-> **Note:** This section assumes familiarity with Linux storage concepts such as attaching and mounting volumes, and [Kubernetes configuration of persistent volume storage.](https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/#create-a-pod)
+In this section, you'll learn how to enable Longhorn to restart workloads after automatically reattaching and remounting volumes.
 
-To enable Longhorn to restart workloads after automatically reattaching and remounting volumes
-
-After reattachment and remount are complete, you may need to manually restart the related workload containers for the volume restoration if the following recommended setup is not applied.
+> This section assumes familiarity with Linux storage concepts such as attaching and mounting volumes, and [Kubernetes configuration of persistent volume storage.](https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/#create-a-pod)
 
 - In a **reattachment,** Longhorn will reattach the volume if the volume engine dies unexpectedly.
 - In a **remount,** Longhorn will detect and remount the filesystem for the volume after the reattachment.
 
+After reattachment and remount are complete, you may need to manually restart the related workload containers for the volume restoration if the following recommended setup is not applied.
 
 ## Requirements
 
-The auto remount does not work for `xfs` filesystem.
+To enable automatically reattaching and remounting volumes, the host filesystem should be `ext4`.
```

Comment:
It's not `the host filesystem`. It's the filesystem on the Longhorn volume.

Reply:
updated